mk/toplevel.mk

```diff
@@ -122,6 +122,32 @@ ifdef TOP_DIR_MKF_LAST
 	@ $(MAKE) test -f $(TOP_DIR_MKF_LAST)
 endif
 
+ifdef TOP_DIR_MKF_FIRST
+FIRST_TEST_CMD=$(MAKE) memcheck -f $(TOP_DIR_MKF_FIRST)
```

Comment:
This can't be called `FIRST_TEST_CMD` - it needs a different name so it doesn't corrupt the `test` target.